src/headers/config.h

```diff
@@ -2,6 +2,7 @@
 // Copyright (C) 2022, Input Labs Oy.
 
 #pragma once
+#include <stdint.h>
```

Comment:
Actually this change is needed, otherwise the build will fail with a more recent compiler, I am using gcc 12.2
````In file included from C:/temp/alpakka_firmware/src/gyro.c:6:
C:/temp/alpakka_firmware/src/headers/config.h:49:5: error: unknown type name 'uint8_t'
   49 |     uint8_t header;
      |     ^~~~~~~
C:/temp/alpakka_firmware/src/headers/config.h:50:5: error: unknown type name 'uint8_t'
   50 |     uint8_t config_version;
      |     ^~~~~~~
C:/temp/alpakka_firmware/src/headers/config.h:51:5: error: unknown type name 'uint8_t'
   51 |     uint8_t os_mode;
      |     ^~~~~~~
C:/temp/alpakka_firmware/src/headers/config.h:52:5: error: unknown type name 'uint8_t'
   52 |     uint8_t profile;
      |     ^~~~~~~
C:/temp/alpakka_firmware/src/headers/config.h:53:5: error: unknown type name 'int8_t'
   53 |     int8_t sensitivity;
      |     ^~~~~~
C:/temp/alpakka_firmware/src/headers/config.h:54:5: error: unknown type name 'int8_t'
   54 |     int8_t deadzone;
      |     ^~~~~~
C:/temp/alpakka_firmware/src/headers/config.h:55:5: error: unknown type name 'int8_t'
   55 |     int8_t touch_threshold;
      |     ^~~~~~
C:/temp/alpakka_firmware/src/headers/config.h:56:5: error: unknown type name 'int8_t'
   56 |     int8_t vibration;
      |     ^~~~~~
C:/temp/alpakka_firmware/src/headers/config.h:65:5: error: unknown type name 'uint8_t'
   65 |     uint8_t padding[256];
      |     ^~~~~~~
C:/temp/alpakka_firmware/src/headers/config.h:70:25: error: unknown type name 'uint8_t'
   70 | void config_set_profile(uint8_t profile);
      |                         ^~~~~~~
C:/temp/alpakka_firmware/src/headers/config.h:7:1: note: 'uint8_t' is defined in header '<stdint.h>'; did you forget to '#include <stdint.h>'?
    6 | #include <stdbool.h>
  +++ |+#include <stdint.h>
    7 |
C:/temp/alpakka_firmware/src/headers/config.h:71:1: error: unknown type name 'uint8_t'
   71 | uint8_t config_get_profile();
      | ^~~~~~~
C:/temp/alpakka_firmware/src/headers/config.h:71:1: note: 'uint8_t' is defined in header '<stdint.h>'; did you forget to '#include <stdint.h>'?
C:/temp/alpakka_firmware/src/headers/config.h:74:1: error: unknown type name 'uint8_t'
   74 | uint8_t config_get_os_mode();
      | ^~~~~~~
C:/temp/alpakka_firmware/src/headers/config.h:74:1: note: 'uint8_t' is defined in header '<stdint.h>'; did you forget to '#include <stdint.h>'?
C:/temp/alpakka_firmware/src/headers/config.h:75:27: error: unknown type name 'uint8_t'
   75 | void config_tune_set_mode(uint8_t mode);
      |                           ^~~~~~~
C:/temp/alpakka_firmware/src/headers/config.h:75:27: note: 'uint8_t' is defined in header '<stdint.h>'; did you forget to '#include <stdint.h>'?
[46/118] Building C object CMakeFiles/alpakka.dir/src/hid.c.obj
ninja: build stopped: subcommand failed.